demos/shapes-custom-question-authoring-demo/src/question/index.js

```diff
@@ -0,0 +1,339 @@
+import { PREFIX, mapSizeStringToFactor } from './constants';
+
+export default class Question {
+    constructor(init, lrnUtils) {
+        this.init = init;
+        this.events = init.events;
+        this.lrnUtils = lrnUtils;
+        this.el = init.$el.get(0);
+
+        this.render().then(() =>{
```

Comment:
spacing after before the "{"
```suggestion
        this.render().then(() => {
```